app.js

```diff
@@ -1,128 +1,55 @@
 'use strict';
 
-var pikeStore = {
-  location: '1st and Pike',
-  openAt: 6, // 6 am
-  closeAt: 20, // 8 pm
-  minCustHr: 23,
-  maxCustHr: 65,
-  avgCookieCust: 6.3,
-  cookiesHrs: [],
-
-  // calculates random number of customers for one hour
-  randomCustHr: function() {
-    return Math.floor(Math.random() * (this.maxCustHr - this.minCustHr + 1) + this.minCustHr);
-  },
-
-  // simulates the number of cookies bought in each hour of a work day
-  simCookiesHrs: function() {
-    this.cookiesHrs = [];
-    var cookiesBought;
-    for (var i = 0; i < (this.closeAt - this.openAt); i++) {
-      cookiesBought = Math.round(this.randomCustHr() * this.avgCookieCust);
-      this.cookiesHrs.push(cookiesBought);
-    }
-  },
-};
+/** Creates a Store with a string for a location name, the minimum customers per hour, maximum cusomers per hour, and average cookies per customer **/
+function Store(location, minCustHr, maxCustHr, avgCookieCust) {
+  this.location = location;
+  this.openAt = 6; // 6 am
+  this.closeAt = 20; // 8 pm
+  this.minCustHr = minCustHr;
+  this.maxCustHr = maxCustHr;
+  this.avgCookieCust = avgCookieCust;
+  this.cookiesHrs = [];
+}
 
-var airportStore = {
-  location: 'SeaTac Airport',
-  openAt: 6, // 6 am
-  closeAt: 20, // 8 pm
-  minCustHr: 3,
-  maxCustHr: 24,
-  avgCookieCust: 1.2,
-  cookiesHrs: [],
-
-  // calculates random number of customers for one hour
-  randomCustHr: function() {
-    return Math.floor(Math.random() * (this.maxCustHr - this.minCustHr + 1) + this.minCustHr);
-  },
-
-  // simulates the number of cookies bought in each hour of a work day
-  simCookiesHrs: function() {
-    this.cookiesHrs = [];
```

Comment:
If this was intentionally added to reset the hourly sales array in the case this function was called multiple times, great thinking.